### ISSUE TITLE
Draw Order

### ISSUE THREAD

**Issue by dbiser99** [raiguard/Factorio-Tapeline#23]
Part questions and part issue...

Is there a way to get the grid to be the base layer? Most things draw on top of it, but tiles and belts (of note...there may be more I haven't noticed yet) draw below it. Tiles I admittedly could see both ways (perhaps leave them below and add an adjustable transparency setting?) but it would be nice to have the belts show up on top of it.

(PS. It seems so obvious I can't help but think it could be something being changed by another mod, but I can't imagine why a mod would change that and admittedly haven't checked.)

**Comment by raiguard** [comment]
Unfortunately I can't do anything about that. I wish belts drew on top but they just don't. The tapes are as "on the ground" as they can get.

I'll ask the devs if it's a bug and if they can fix it.

**Comment by dbiser99** [comment]
Thanks for letting me know. I like the nearly black background, but will play around with the alpha channel to adjust the transparency a little instead then. (It was actually pretty obvious, but I hadn't registered the values as being in hex when I wrote the first comment asking for a way to adjust the transparency. Once I did though I knew exactly what I was looking at. That said I don't know if there might be a way to better explain that for someone who might not know and/or to use the 0-255 numeric values instead and let the mod code do the conversion to hex if it needs to.)
